### ISSUE TITLE
Enabling hibernate trace logging crashes context initialization - query can't be created (UnknownFormatConversionException)

### ISSUE THREAD

**Issue by mat-mik** [spring-projects/spring-data-jpa#3068]
Hi team,

after upgrading to Spring Boot 3 I experienced an issue when enabling `org.hibernate` logging level on `trace` / `debug`.

Example: https://github.com/mat-mik/spring-jpa-logging-error

An example has one entity and one repository with custom `@Query` method which uses  `like escaping` from https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#jpa.query.spel-expressions

Stacktrace:
```
Caused by: org.springframework.data.repository.query.QueryCreationException: Could not create query for public abstract java.util.List com.example.springjpaloggingerror.SomeRepository.findByQuery(java.lang.String); Reason: Validation failed for query for method public abstract java.util.List com.example.springjpaloggingerror.SomeRepository.findByQuery(java.lang.String)
// ...
Caused by: java.lang.IllegalArgumentException: Validation failed for query for method public abstract java.util.List com.example.springjpaloggingerror.SomeRepository.findByQuery(java.lang.String)
// ...
Caused by: java.util.UnknownFormatConversionException: Conversion = '''
```

Some workaround:
https://github.com/mat-mik/spring-jpa-logging-error/blob/main/src/test/resources/application.properties#L4

___
I just realised that enabling `debug` also causes the same issue.

**Comment by schauder** [comment]
When Hibernates logging creates exceptions that would be a Hibernate issue. Please report this with their issue tracker https://hibernate.atlassian.net/jira/projects

**Comment by mat-mik** [comment]
For those experiencing the same error - fixed in Hibernate 6.2.6
